examples/addLiquidityNested.ts

```diff
@@ -102,8 +101,7 @@ const addLiquidityNested = async () => {
     const { call, to, value, minBptOut } = addLiquidityNested.buildCall({
         ...queryOutput,
         slippage,
-        sender: accountAddress,
-        recipient: accountAddress,
+        accountAddress,
```

Comment:
Are we loosing the option for users to set sender/receiver and is that ok? If yes should this not be made consistent for `addLiquidity` too?

Reply:
Talked to Daniel about this and he's going to check with the SC team on how to approach sender/recipient on v3. There's a chance they will update the interface on their end, so we'll leave as is for now and get back to it later if needed.
Nested liquidity V2 is a special case due to the gas cost reduction code that handles transactions between user and relayer, so we'll keep a single `accountAddress` as sender/recipient